src/rpc.ts

```diff
@@ -1,15 +1,28 @@
 import express from 'express';
 import snapshot from '@snapshot-labs/strategies';
-import scores, { blockNumByNetwork } from './scores';
+import scores from './scores';
 import redis from './redis';
-import { clone, sha256, formatStrategies, rpcSuccess, rpcError } from './utils';
+import {
+  clone,
+  sha256,
+  formatStrategies,
+  rpcSuccess,
+  rpcError,
+  getBlockNum,
+  blockNumByNetwork
+} from './utils';
 import { version } from '../package.json';
 
 const router = express.Router();
 
 router.post('/', async (req, res) => {
   const { id = null, params = {} } = req.body;
   try {
+    typeof params.snapshot != 'number' && (params.snapshot = 'latest');
```

Comment:
What's this?

Reply:
If anyone passes something other than a block number, we now use `'latest'`. we don't need it?

Reply:
This is hard to read I would rather do
```
if (typeof params.snapshot !== 'number') params.snapshot = 'latest';
```